core/renderers/zelos/constants.js

```diff
@@ -380,6 +402,26 @@ Blockly.zelos.ConstantProvider.prototype.init = function() {
       this.INSIDE_CORNERS.rightWidth;
 };
 
+/**
+ * @override
+ */
+Blockly.zelos.ConstantProvider.prototype.setTheme = function(theme) {
+  Blockly.zelos.ConstantProvider.superClass_.setTheme.call(this, theme);
+
+  var selectedGlowColour = theme.getComponentStyle('selectedGlowColour');
+  this.SELECTED_GLOW_COLOUR = selectedGlowColour != null ?
+      selectedGlowColour : this.SELECTED_GLOW_COLOUR;
+  var selectedGlowSize = theme.getComponentStyle('selectedGlowSize');
+  this.SELECTED_GLOW_SIZE = selectedGlowSize != null ?
+      selectedGlowSize : this.SELECTED_GLOW_SIZE;
+  var replacementGlowColour = theme.getComponentStyle('replacementGlowColour');
+  this.REPLACEMENT_GLOW_COLOUR = replacementGlowColour != null ?
+      replacementGlowColour : this.REPLACEMENT_GLOW_COLOUR;
```

Comment:
Nit and not necessary but might look cleaner to change to replacementGlowColour || this.REPLACEMENT_GLOW_COLOUR